test/components/Product/Product.spec.ts

```diff
@@ -150,13 +159,13 @@ describe("Product", () => {
 
     it("should pick n-sku-selector change events", () => {
       element.append(
-        <select n-sku-selector>
-          <option value="456">SKU 1</option>
-          <option value="457" selected>
-            SKU 2
-          </option>
-        </select>,
-        <div n-atc>Add to cart</div>
+        createElement(
```

Comment:
add createElements that takes multiple varargs html expressions and returns an array of elements